include/mongoose.hrl

```diff
@@ -34,3 +34,11 @@
 %% ---------------------------------
 %% Logging mechanism
 -include("mongoose_logger.hrl").
+
+%% for worker_pool
+-record(mongoose_worker_pool, {name :: atom(),
```

Comment:
I'd move it to separate header file. Not that many modules need this definition anyway.